.github/workflows/main.yml

```diff
@@ -1988,6 +1988,9 @@ jobs:
       - name: enterprise/j2ee.core
         run: ant $OPTS -f enterprise/j2ee.core test
 
+      - name: enterprise/j2ee.core
+        run: ant $OPTS -f enterprise/maven.j2ee test
```

Comment:
you forgot to edit the name
` - name: enterprise/maven.j2ee`

Reply:
Oups. Done!